dev_guide/builds.adoc

```diff
@@ -2529,15 +2532,42 @@ list of custom labels that will be applied to each image built from the BuildCon
 ====
 [source,yaml]
 ----
-output:
-  to:
-    kind: "ImageStreamTag"
-    name: "my-image:latest"
-  imageLabels:
-  - name: "vendor"
-    value: "MyCompany"
-  - name: "authoritative-source-url"
-    value: "registry.mycompany.com"
+spec:
+  output:
+    to:
+      kind: "ImageStreamTag"
+      name: "my-image:latest"
+    imageLabels:
+    - name: "vendor"
+      value: "MyCompany"
+    - name: "authoritative-source-url"
+      value: "registry.mycompany.com"
+----
+====
+
+[[output-image-digest]]
+=== Output Image Digest
+
+Built images can be uniquely identified by their
+link:++https://docs.docker.com/registry/spec/api/#/content-digests++[digest], which can
```

Comment:
@mmilata Interesting. I've not seen that before. @adellape @vikram-redhat Do you know a workaround, or do you think the ++ can be accepted? Not sure if this will impact how the docs build. Thanks!